src/client/layouts/MinimalLayout.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
The previous `Main` layout had a number of stories detailing all the different ways you could use this component. It's probably worth adding those back in, so it's easier to see how the new `MininalLayout` can be composed.

Reply:
Good idea.